pastas/model.py

```diff
@@ -1253,8 +1253,31 @@ def get_transform_contribution(self, tmin=None, tmax=None):
         sim_org = ml.simulate(tmin=tmin, tmax=tmax)
         return sim - sim_org
 
+    def get_response(self, rfunc, name, parameters=None, dt=None, **kwargs):
```

Comment:
use of rfunc as variable for block or step is a bit confusing here?

Reply:
How elaborate are our docstrings for internal methods? And do we hide internal methods by using the prefix _?

Reply:
This is an internal method. I would love to change all internal methods to the method name preceded by a single or double underscore but this is another issue #74 which should not be addressed in this PR.

Reply:
maybe just the more general `func`? Can't think of anything better at the moment.

Reply:
changed the argument name to "block_or_step" which is a little long but more explicit. Also added a docstring for the method.